server/api/sport/sport.controller.js

```diff
@@ -72,9 +72,10 @@ exports.destroy = function(req, res) {
 exports.populate = function(req, res) {
     Sport.find(req.params.id).then(function (sport) {
         if(!sport) { return res.send(404); }
-        parseCsv(sport)
+        return parseCsv(sport)
+    }).then(function() {
         return res.send(201, sport);
-    }, function(error){
+    }.catch(function(err) {
```

Comment:
There's other segments in the controller that haven't swapped to using .catch. Should we change all of them to use this format, or would you rather keep it the way it is?